### PR TITLE
handle Describedby in iframe and multiple Describedby IDs

### PR DESCRIPTION
**Pull request by yalingliu2**
Fixed issue #5784: IE11: aria-describedby on buttons inside iFrames not respected. In addition, changed the code to support aria-describedby containing multiple ids using a space separated list.

### CODE REVIEW

File: source/virtualBuffers/MSHTML.py [hunk 1]
```diff
@@ -76,15 +76,25 @@ def _normalizeControlField(self,attrs):
...
+			ariaDescribedByIds=ariaDescribedBy.split(" ")
+			for ariaDescribedById in ariaDescribedByIds:
+				descNode=None				
 				try:
-					description=self.obj.makeTextInfo(NVDAObjects.IAccessible.MSHTML.MSHTML(HTMLNode=descNode)).text
-				except:
+					descNode=self.obj.rootNVDAObject.HTMLNode.document.getElementById(ariaDescribedBy)
+				except (COMError,NameError):
+					descNode=None	
+				if descNode:
```
Comment (michaelDCurran) [inline source/virtualBuffers/MSHTML.py]: Rather than if descNode: pass else: please change it to simply just if not descNode:

**Comment by yalingliu2** [comment]
I tried if not descNode, also tried if descNode is None, NVDA doesn't read the specified describedby label of buttons in either iframe or non-iframe in browse mode. I am running from source, using IE11 and windows 10. if !descNode works, although I am not sure whether it's the correct Python syntax.

**Comment by michaelDCurran** [comment]
No. That would just give a syntax error.

**Comment by michaelDCurran** [comment]
Replaced by #6738.

**Pull request closed without merging**